src/components/Modal/ModalFooter/ModalFooterButtons/ModalFooterButtons.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { forwardRef } from "react";
+import VibeComponentProps from "../../../../types/VibeComponentProps";
+import VibeComponent from "../../../../types/VibeComponent";
+import { getTestId } from "../../../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../../../tests/constants";
+import { NOOP } from "../../../../utils/function-utils";
+import Flex from "../../../Flex/Flex";
+import Button from "../../../Button/Button";
+import ModalFooter from "../ModalFooter";
+
+export interface ModalFooterButtonsProps extends VibeComponentProps {
+  primaryButtonText?: string;
+  secondaryButtonText?: string;
+  onPrimaryButtonClick?: () => void;
+  onSecondaryButtonClick?: () => void;
+}
+
+const ModalFooterButtons: VibeComponent<ModalFooterButtonsProps> = forwardRef(
+  (
+    {
+      primaryButtonText = "Confirm",
+      secondaryButtonText = "Cancel",
```

Comment:
since texts are always translated, I suggest removing those default props as it can cause confusion in terms of usage.

Reply:
good point 💪🏼